# gets the active dof values of the robot

int32 bodyid

# specifies indices into the active dofs to get. if empty, returns all active dof values
uint8[] indices

---
float32[] values
